jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -0,0 +1,811 @@
+---
+title: "Version 1.6.0 released"
+author: The Nim Team
+---
+
+We are very proud to announce Nim version 1.6, the latest stable release, after a year in the making!
+This is by far the biggest release yet, here are some stats:
+* 1667 PRs merged (1760 commits)
+* 893 issues closed
+* 15 new stdlib modules
+* new features in >40 stdlib modules, including major improvements to 10 commonly used modules.
+* documentation and minor improvements to 170 modules, including 312 new runnable examples
+* 280 new nimble packages
+
+Nim made its first entry in TIOBE index in 2017 at position 129,
+last year it entered the top-100, and last 2 months it entered the top-50 (https://forum.nim-lang.org/t/8297).
+We hope this release will reinforce this trend, building on Nim's core strenghs:
+a practical, compiled systems programming language; offering C-like performance and portability;
+Python-like syntax; LISP-like flexibility; strong C, C++, JS, python interop;
+and best-in class metaprogramming.
+
+This release includes improvements in the following areas:
+* new language features (user defined literals, private imports, strict effects, `iterable[T]`, new style concepts, dot-like operators, block arguments with optional params)
+* new compiler features (`nim --eval:cmd`, custom nimscript extensions, customizable compiler messages)
+* major improvements to `--gc:arc`, `--gc:orc`
+* correctness and performance of integer and float parsing and rendering in all backends
+* significant improvements in error messages, showing useful context
+* doc generation logic and documentation, in particular `runnableExamples` now works in more contexts and replaces `code-block`.
+* made JS, VM and nimscript backend more consistent with C backend, allowing more modules to work with those backends, including the imports from `std/prelude`; the test suite now standardizes on testing stdlib modules on each major backend (C, JS, VM).
+* support for Apple silicon/M1, 32-bit RISC-V, improved support for NodeJS backend
```

Comment:
@Araq should i add CROSSOS to this list (refs https://github.com/nim-lang/Nim/pull/18889)

Reply:
ended up adding it